addons/movments_accounts_balances/controllers/controllers.py

```diff
@@ -79,3 +97,122 @@ def general_ledger_report(self, account_id, start_date, end_date):
             'ledger_data': ledger_data,
         }
 
+    @api.model
+    def get_bill(self, bill_id):
+        # Define search criteria to filter bills
+        domain = [
+            ('id', '=', bill_id),
+            ('move_type', '=', 'in_invoice'),
+        ]
+
+        # Retrieve bills based on the criteria
+        bills = self.env['account.move'].search(domain, order='invoice_date')
+
+        # Prepare a list to store bill data
+        bill_data = []
+
+        for bill in bills:
```

Comment:
@sameerz09 there would never be 2 bills with same ID? You should always expect just one bill similar to `delete_bill`